rama-net/src/address/host.rs

```diff
@@ -18,6 +18,25 @@ pub enum Host {
     Address(IpAddr),
 }
 
+impl Host {
+    /// Local loopback address (IPv4)
+    pub const LOCALHOST_IPV4: Self = Self::Address(IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)));
+
+    /// Local loopback address (IPv6)
+    pub const LOCALHOST_IPV6: Self = Self::Address(IpAddr::V6(Ipv6Addr::new(0, 0, 0, 0, 0, 0, 0, 1)));
+
+    /// Local loopback name
+    pub const LOCALHOST_NAME: Self = Self::Name(Domain::from_static("localhost"));
+
+    /// Default address, not routable
+    pub const DEFAULT_IPV4: Self = Self::Address(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)));
+
+    /// Broadcast address (IPv4)
+    pub const BROADCAST_IPV4: Self = Self::Address(IpAddr::V4(Ipv4Addr::new(255, 255, 255, 255)));
+
+    pub const EXAMPLE: Self = Self::Name(Domain::from_static("example.com"));
```

Comment:
Given you already have a convention established here it follows that this is probably better named `EXAMPLE_NAME`